lib/standalone.mjs

```diff
@@ -116,6 +100,24 @@ export default async function standalone({
 		quietDeprecationWarnings,
 	});
 
+	/** @type {Formatter} */
+	let formatterFunction;
+
+	try {
+		/** @see https://github.com/stylelint/stylelint/issues/7447 */
+		if (!quietDeprecationWarnings && formatter === 'github') {
+			emitDeprecationWarning(
+				'"github" formatter is deprecated.',
+				'GITHUB_FORMATTER',
+				'Please use "stylelint-actions-formatters" instead.',
+			);
+		}
```

Comment:
[suggest] Can you move this `emitDeprecationWarning()` into `getFormatter()`?

Reply:
I tried to but getFormatter can be called more than one time in certain cases so the message was printed more than once. Tried to keep it from happening but in vain. Any idea?

Reply:
Okay, let's keep it as-is.